lib/lookup.json

```diff
@@ -171,7 +174,8 @@
   "express-session": {
     "prefix": "v",
     "maintainers": "dougwilson",
-    "skip": "win32"
+    "skip": ">=16",
+    "comment": "https://github.com/expressjs/session/pull/844"
   },
   "fastify": {
```

Comment:
I'm unsure why Fastify is passing given https://github.com/fastify/fastify/pull/3456 and https://github.com/nodejs/node/pull/40858

Reply:
fastify is not passing. this PR is not exhaustive